src/topology/algebra/polynomial.lean

```diff
@@ -203,6 +203,31 @@ begin
     rw [←hcd, noroots, card_zero], },
 end
 
+lemma coeff_bdd_of_roots_le (B : ℝ) (d : ℕ) (f : F →+* K) :
+   ∃ C, ∀ p : F[X], p.monic → splits f p → p.nat_degree ≤ d → (∀ z ∈ (map f p).roots, ∥z∥ ≤ B) →
+   ∀ i, ∥(map f p).coeff i∥ ≤ C :=
+ begin
+   let S := finset.bUnion (finset.product (finset.range (d + 1)) (finset.range (d + 1)))
+     (λ x, ( { B ^ (x.1 - x.2) * (x.1.choose x.2) } : finset ℝ)),
+   have hS : S.nonempty,
+   { exact finset.bUnion_nonempty.mpr
+       ⟨⟨0 , 0⟩, finset.mem_product.mpr ⟨finset.mem_range_succ_iff.mpr (zero_le _),
+         finset.mem_range_succ_iff.mpr (zero_le _)⟩, finset.singleton_nonempty _⟩, },
+   let C := (S.max' hS),
+   { use max C 0,
```

Comment:
I can be wrong, but the indentation here is not needed, right? There is only one goal I think (I don't have VS Code opened...).

Reply:
Indeed, indentations were all wrong in this function. Thanks!